crates/common/tedge_config/src/system_services/managers/config.rs

```diff
@@ -4,10 +4,12 @@ use std::fs;
 use std::path::PathBuf;
 
 pub const SERVICE_CONFIG_FILE: &str = "system.toml";
+const INIT_COMMAND: &[&str] = &["init", "6"];
```

Comment:
This name is confusing as there is also an unrelated `InitConfig` struct.

```suggestion
const REBOOT_COMMAND: &[&str] = &["init", "6"];
```